core/test/download.test.ts

```diff
@@ -124,6 +124,22 @@ test('can download text by keeping eyes on abort signal', async () => {
   await rejects(response2, (e: Error) => e.name === 'AbortError')
 })
 
+test('detect unpopular encode type', async () => {
+  let task = createDownloadTask()
+
+  expectRequest('https://example.com').andRespond(
+    200,
+    'Hi',
```

Comment:
This test will pass even if we haven’t encoding feature.

What do you think if we will use CP1251 and put that broken (when CP1251 was parsed as UTF-8) encoding text here?

Reply:
i will check later

Reply:
hm, if i just cope paste broken text (like �), it doesnt work correctly. Continue research

Reply:
Try binary format for symbol

Reply:
Another way is to get JS code for `UTF-8→CP1251` convertation and put:

```js
to1251('тест')
```

Reply:
sorry, what do you mean "Try binary format for symbol"?

Reply:
> sorry, what do you mean "Try binary format for symbol"?

[String.fromCodePoint](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/fromCodePoint)